### DISCUSSION TITLE
The help is completely brocken (nothing shows up on the right-hand side)

### DISCUSSION THREAD

**Discussion by tozGitHub** [WinEdt-Team/WinEdt-Support#23] [2.2 Issues, Bugs and Problems]
Dear Alex, dear community,

It seems that the help is brocken. The table of the contents on the left-hand side shows up, but nothing on the right-hand side:

![image](https://github.com/WinEdt-Team/WinEdt-Support/assets/52357589/7235495d-d9c1-4367-8e12-8988df56f296)

The problem seems to exist regardless whether I try to display the content by the table of contents, the index, or the search.
Also it does not matter whether I simply click on the elements, let us say, in the table of contents or double click on them.

WinEdt version: Build: 20240317  (v. 11.2) - 64-bit
Windows version: 10.0.22631 Build 22631.

Best,
Thomas

**Comment by Alex-Simonic** [comment]
The help is fine. It appears that Windows does not like this file to be unzipped for security reasons:

![image](https://github.com/WinEdt-Team/WinEdt-Support/assets/157834149/e8c56daa-3fed-40b7-aa31-fd114ded071b)

Now you know what to do.
